account/accounts_nimbus.go

```diff
@@ -6,23 +6,24 @@ import (
 	"github.com/status-im/status-go/account/generator"
 )
 
-// NewManager returns new node account manager.
-func NewManager() *Manager {
-	m := &Manager{}
-	m.accountsGenerator = generator.New(m)
+// NimbusManager represents account manager interface.
+type NimbusManager struct {
+	*Manager
+}
+
+// NewNimbusManager returns new node account manager.
+func NewNimbusManager() *NimbusManager {
+	m := &NimbusManager{}
+	m.Manager = &Manager{accountsGenerator: generator.New(m)}
 	return m
 }
 
-// InitKeystore sets key manager and key store.
-func (m *Manager) InitKeystore(keydir string) error {
-	m.mu.Lock()
-	defer m.mu.Unlock()
+// // InitKeystore sets key manager and key store.
+// func (m *Manager) InitKeystore(keydir string) error {
```

Comment:
Please don't commit commented out stuff like this.

Reply:
This is kind of a special case, where we're committing a partially implemented feature that's hidden behind a feature flag. I'm happy to discuss it on a call. Maybe you'd like to join the handover call that I'll be having with @andremedeiros?